File: problem_twenty_eight.py
# Character check 
inp =input("Enter character:")
def Case_Check(inp):
    if inp >= "0" and inp <= "9":
        print("Numberical Character.")
    elif inp >= "a" and inp <="z":
        print("Lower Case Character.")
    elif inp >= "A" and inp <="Z":
        print("Upper Case Character.")
    else:
        print("Special Char")

Case_Check(inp)                    

File: problem_twenty_eight.js
// Character check 
var inp = a
function myFunction(inp){
    if (inp >="a"&& inp<="z"){
         console.log("Lowercase Character")
      }else if(inp >="A" && inp <="Z"){
        console.log("Uppercase Character")   
      }else if(inp >=0 && inp <=9){
        console.log("Numberical Character")
      }else{
        console.log("Speial Character")
      }
}
myFunction(inp)


